.github/scripts/README.md

```
@@ -0,0 +1,269 @@
+gh api repos/OWNER/REPO/pulls/123 | python .github/scripts/context_chunker.py
+
+# Test without tiktoken (heuristic mode)
+pip uninstall -y tiktoken
+python .github/scripts/context_chunker.py
+```
+
+**Note**: The script will automatically detect if `tiktoken` is available and use the appropriate estimation method.
+
+## Benefits
+
+### Prevents Errors
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)